clock/rfc822_test.go

```diff
@@ -67,7 +67,7 @@ func TestRFC822Unmarshaling(t *testing.T) {
 		// Extrapolating the numerical offset from an abbreviated offset is unreliable. In
 		// this test case the RFC3339 will have the incorrect result due to limitation in
 		// Go's time parser.
-		outRFC3339: "2019-08-29T11:20:07Z",
+		outRFC3339: "2019-08-29T11:20:07+03:00",
```

Comment:
The comment above is not longer relevant then, and should be removed.

Reply:
Running these tests locally for me are failing.  I'm running go 1.17.1. It looks like it is because go parses the time to your local time zone. So for the tests to pass for me, I had to set my computer's timezone to `MSK`. 

https://stackoverflow.com/questions/25368415/how-to-properly-parse-timezone-codes 

```
expected: "2019-08-29T11:20:07+03:00"
actual  : "2019-08-29T11:20:07Z"
```